EnhancedMediaPlayer/MediaPlayerViewController.swift

```diff
@@ -55,11 +71,18 @@ public class MediaPlayerViewController: UIHostingController<MediaPlayerView> {
     }
     
     private func rewind() {
-        // TODO: implement rewind
+        seek(forward: false)
     }
     
     private func forward() {
-        // TODO: implement forward
+        seek(forward: true)
+    }
+
+    private func seek(forward: Bool) {
+        let current = viewModel.player.currentTime().seconds
+        let target: Double = forward ? current + viewModel.seekFactor : current - viewModel.seekFactor
```

Comment:
Could be
```swift
let current = viewModel.player.currentTime().seconds
let seekTimeToApply = forward ? viewModel.seekFactor : -viewModel.seekFactor
let target = current + seekTimeToApply
```
Your call though, choose whatever you think is more readable

Reply:
I like it is more concise, but don't really like this `current + (-seekFactor)` math equation